test/blackbox/common/RTPSBlackboxTestsBasic.cpp

```diff
@@ -1182,6 +1182,101 @@ TEST(RTPS, participant_ignore_local_endpoints_two_participants)
     eprosima::fastrtps::rtps::RTPSDomain::removeRTPSParticipant(participant_reader);
 }
 
+/* Maximum number of bytes allowed for an RTPS datagram generated by this participant. */
+TEST(RTPS, max_output_message_size_participant)
```

Comment:
After reviewing the documentation I realized it would be nice to duplicate this test in `DDSBlackboxTestsBasic.cpp`